packages/adobe_analytics/android/src/main/kotlin/be/vrt/adobe_analytics/AdobeAnalyticsPlugin.kt

```diff
@@ -22,6 +24,16 @@ class AdobeAnalyticsPlugin : MethodCallHandler {
         val exhaustMe: Any = when (val data = Extractor.adobeCallFromCall(call)) {
             is Extractor.AdobeCall.TrackState -> trackState(data, result)
             is Extractor.AdobeCall.TrackAction -> trackAction(data, result)
+            is Extractor.AdobeCall.GetExperienceCloudId -> {
+                val mainLooper = Handler()
+                Thread().run {
```

Comment:
Not sure how this `Thread` will be GC'd after use.... 🤔

Reply:
Verified GC with @petermnt  ; works as intended 👍